nautobot/extras/migrations/0069_move_cf_name_value_to_label.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.2.18 on 2023-03-14 15:45
+
+from django.db import migrations
+
+
+def move_cf_name_value_to_label(apps, schema_editor):
+    CustomField = apps.get_model("extras", "customfield")
+    # Filtering on null or empty labels
+    custom_fields = CustomField.objects.filter(label__isnull=True).union(CustomField.objects.filter(label__exact=""))
+
+    for cf in custom_fields:
+        cf.label = cf.name
+        cf.save()
+
```

Comment:
Aren't we missing somewhere a data migration that will iterate over all CustomFieldModels and migrate `self._custom_field_data[cf.name]` to `self._custom_field_data[cf.key]` across the board?